### PR TITLE
Adding confirmation dialog for clear all the items

### PR DESCRIPTION
**Pull request by b00f**
Add confirmation dialog to the extension.

Addressing #182

### CODE REVIEW

File: extension.js [hunk 1]
```diff
@@ -332,22 +333,27 @@ const ClipboardIndicator = Lang.Class({
...
-            if (!mItem.currentlySelected) {
-                let idx = that.clipItemsRadioGroup.indexOf(mItem);
-                mItem.destroy();
-                that.clipItemsRadioGroup.splice(idx,1);
-            }
+        const title = _("Clear all?");
+        const message = _("Are you sure you want to delete all clipboard items?");
+        const sub_message = _("This operation cannot be undone.");
+
+        ConfirmDialog.openConfirmDialog(title, message, sub_message, _("Clear"), () => {;
```
Comment (RaphaelRochet) [inline extension.js:341]: I wonder .. why did you use consts for all strings but ok_label ?

File: confirmDialog.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+const St = imports.gi.St;
+const GObject = imports.gi.GObject;
+const ModalDialog = imports.ui.modalDialog;
+const CheckBox = imports.ui.checkBox;
+const Clutter = imports.gi.Clutter;
+
+const Gettext = imports.gettext.domain("gnome-trash");
```
Comment (RaphaelRochet) [inline confirmDialog.js:7]: You forgot to edit this ? We should use extension context

**Comment by RaphaelRochet** [comment]
A shame that GS does not provide a simple confirm() function ...
A minimum of tweaking could make this dialog perfect !
Here is a side-by-side with a gedit confirm dialog to help :
image

File: confirmDialog.js (hunk 2 above)
Comment (b00f) [reply confirmDialog.js:7]: Good find. Copy-pasting issue. Thanks man!

**Comment by b00f** [comment]
> A minimum of tweaking could make this dialog perfect !

Can you explain what do you mean? You mean make it wider?

**Comment by RaphaelRochet** [comment]
No only tweak font size and spacings. Very minor things ... :smile: 
Do you think there's a "right" way to use color on "Clear" button ? I mean without breaking themes ...
Like this dialog in nautilus : 
image

File: extension.js (hunk 1 above)
Comment (b00f) [reply extension.js]: Done! As requested.

File: confirmDialog.js [hunk 3]
```diff
@@ -4,19 +4,19 @@ const ModalDialog = imports.ui.modalDialog;
 const CheckBox = imports.ui.checkBox;
 const Clutter = imports.gi.Clutter;
 
-const Gettext = imports.gettext.domain("gnome-trash");
+const Gettext = imports.gettext.domain("clipboard-indicator");
 const _ = Gettext.gettext;
```
Comment (RaphaelRochet) [inline confirmDialog.js:9]: You can remove these now. This will make this confirm-dialog class reusable without a change :+1:

Comment (b00f) [reply confirmDialog.js:9]: Unfortunately the translation won't work if I remove it. I don't like it as well.

**Comment by b00f** [comment]
We can modify CSS file, But I prefer to not. Keep it as it is. Gnome is different from distro to distro.

File: confirmDialog.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+      let subject_label = new St.Label({
+        style: `font-weight: 700`,
+        text: title
+      });
+
+      message_box.add(subject_label, { y_fill: true, y_align: St.Align.START });
+
+      let desc_label = new St.Label({
+        style: 'padding-top: 10px; padding-bottom: 20px;',
```
Comment (RaphaelRochet) [inline confirmDialog.js:41]: Can't argue with theming, but at least remove double bottom space, and follow gnome guidelines on element spacing :
```suggestion
        style: 'padding-top: 12px; ',
```

File: confirmDialog.js (hunk 3 above)
Comment (RaphaelRochet) [reply confirmDialog.js:9]: I just update the French translation with these strings, and removed these 2 lines : works like a charm :)

Comment (b00f) [reply confirmDialog.js:9]: How about `Gettext.gettext`? Do you import gettext?

Comment (RaphaelRochet) [reply confirmDialog.js:9]: I removed both lines. There's no use for "_" in that file.

File: confirmDialog.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
+const St = imports.gi.St;
+const GObject = imports.gi.GObject;
+const ModalDialog = imports.ui.modalDialog;
+const CheckBox = imports.ui.checkBox;
+const Clutter = imports.gi.Clutter;
+
+const Gettext = imports.gettext.domain("clipboard-indicator");
+const _ = Gettext.gettext;
```
Comment (RaphaelRochet) [inline confirmDialog.js:8]: Like said.
```suggestion
```

**Pull request merged**